src/sage/plot/complex_plot.pyx

```diff
@@ -731,21 +733,25 @@ def add_contours_to_rgb(rgb, delta, dark_rate=0.5):
 
     EXAMPLES::
 
-        sage: import numpy as np
-        sage: from sage.plot.complex_plot import add_contours_to_rgb
-        sage: add_contours_to_rgb(np.array([[[0, 0.25, 0.5]]]), np.array([[0.75]]))  # abs tol 1e-4
+        sage: # needs numpy
+        sage: import numpy as np                                                        # needs numpy
+        sage: from sage.plot.complex_plot import add_contours_to_rgb                    # needs numpy
```

Comment:
We have `# needs numpy`  for both a block scope and for the lines. Remove on the lines